content/tutorial-checks.md

```diff
@@ -1,588 +1,928 @@
-                     "OwnerCount" : 1,
-                     "Sequence" : 2
-                  },
-                  "PreviousTxnID" : "5168CE1F7FB10EED87E6DD94DAE5DD910B120869E5DCE682F6995BDAEA40DACA",
-                  "PreviousTxnLgrSeq" : 5
-               }
-            }
-         ],
-         "TransactionIndex" : 0,
-         "TransactionResult" : "tesSUCCESS"
-      },
-      "status" : "success",
-      "validated" : true
-   }
-}
+*WebSocket*
+
+```json
+{% include 'code_samples/checks/websocket/sign-create-req.json' %}
+```
+
+*Commandline*
+
+```bash
+{% include 'code_samples/checks/cli/sign-create-req.sh' %}
 ```
 
-find CreatedNode of type Check, LedgerIndex is its ID: `49647F0D748DC3FE26BDACBC57F251AADEFFF391403EC9BF87C97F67E9977FB0`
+<!-- MULTICODE_BLOCK_END -->
+
+#### Example Response
+
+<!-- MULTICODE_BLOCK_START -->
 
-## Cancel Check
+*RippleAPI*
 
+```js
+{% include 'code_samples/checks/js/sign-create-resp.txt' %}
 ```
-$ ./rippled submit snkuWqxoqt6aeykTbkEWrTMJHrWGM '{
->     "Account": "rUn84CUYbNjRoTQ6mSW7BVJPSVJNLb1QLo",
->     "TransactionType": "CheckCancel",
->     "CheckID": "49647F0D748DC3FE26BDACBC57F251AADEFFF391403EC9BF87C97F67E9977FB0",
->     "Fee": "12"
-> }'
-Loading: "/home/mduo13/.config/ripple/rippled.cfg"
-2018-Jan-24 01:11:07 HTTPClient:NFO Connecting to 127.0.0.1:5005
 
-{
-   "result" : {
-      "engine_result" : "tesSUCCESS",
-      "engine_result_code" : 0,
-      "engine_result_message" : "The transaction was applied. Only final in a validated ledger.",
-      "status" : "success",
-      "tx_blob" : "12001222800000002400000003501849647F0D748DC3FE26BDACBC57F251AADEFFF391403EC9BF87C97F67E9977FB068400000000000000C7321022C53CD19049F32F31848DD3B3BE5CEF6A2DD1EFDA7971AB3FA49B1BAF12AEF78744630440220615F9D19FA182F08530CD978A4C216C8676D0BA9EDB53A620AC909AA0EF0FE7E02203A09CC34C3DB85CCCB3137E78081F8F2B441FB0A3B9E40901F312D3CBA0A67A181147990EC5D1D8DF69E070A968D4B186986FDF06ED0",
-      "tx_json" : {
-         "Account" : "rUn84CUYbNjRoTQ6mSW7BVJPSVJNLb1QLo",
-         "CheckID" : "49647F0D748DC3FE26BDACBC57F251AADEFFF391403EC9BF87C97F67E9977FB0",
-         "Fee" : "12",
-         "Flags" : 2147483648,
-         "Sequence" : 3,
-         "SigningPubKey" : "022C53CD19049F32F31848DD3B3BE5CEF6A2DD1EFDA7971AB3FA49B1BAF12AEF78",
-         "TransactionType" : "CheckCancel",
-         "TxnSignature" : "30440220615F9D19FA182F08530CD978A4C216C8676D0BA9EDB53A620AC909AA0EF0FE7E02203A09CC34C3DB85CCCB3137E78081F8F2B441FB0A3B9E40901F312D3CBA0A67A1",
-         "hash" : "414558223CA8595916BB1FEF238B3BB601B7C0E52659292251CE613E6B4370F9"
-      }
-   }
-}
+*WebSocket*
+
+```json
+{% include 'code_samples/checks/websocket/sign-create-resp.json' %}
+```
+
+*Commandline*
+
+```json
+{% include 'code_samples/checks/cli/sign-create-resp.txt' %}
 ```
 
-## wait for validation / ledger_accept
+<!-- MULTICODE_BLOCK_END -->
 
-## tx on checkcancel
+### {{send_n.next()}}. Submit the signed transaction
```

Comment:
To beat what may at this point already be a dead horse, the text in the include reads more like a reference than a tutorial. Again, as a user, I might ask something like, "OK, cool. What do _I_ do now?"